script.js

```diff
@@ -38,32 +46,23 @@ function compare(a, b) {
     return comparison;
 }
 
-
-/////////////////// *********           Episodes Section           *********////////////////
-
-
+// loop on all episodes to draw the page DOM
 function makePageForEpisodes(episodeList) {
-    const rootElem = document.getElementById("root");
+    
     episodeList.forEach(episode => {
         setBoxEpisode(episode);
     });
-    //rootElem.textContent = `Got ${episodeList.length} episode(s)`;
 }
 
+// prepare the episode selector
 function setupSelector(episodeList){
     let episodeSelect = document.getElementById("episodeSelect");
     episodeList.forEach(episode => {
         setSelectorEpisode(episode, episodeSelect);
     });
-
-    const allShows = getAllShows();
-    allShows.sort(compare);
-    let showSelect = document.getElementById("showsSelect");
```

Comment:
you can add an Option with "All Shows" text before adding shows name.

Reply:
Thank you Summaya for your comments, I've fixed them and uploaded a new version.